keps/sig-network/1435-mixed-protocol-lb/README.md

```diff
@@ -476,6 +476,20 @@ Kube-proxy will not block traffic based on the port status information from `Ser
 
 ### Test Plan
 
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+##### Unit tests
+
+Proper unit tests are in place for the affected code parts.
```

Comment:
Can you link to  some of the existing tests? (I think that's a requirement for the new test plan format)

Reply:
https://github.com/kubernetes/kubernetes/blob/867b5cc31b376c9f5d04cf9278112368b0337104/pkg/registry/core/service/strategy_test.go#L315 
https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/pkg/apis/core/validation/conditional_validation_test.go#L29

Reply:
Thank you for the links, fixed.